sdk/core/azure_core_tracing_opentelemetry/README.md

```diff
@@ -0,0 +1,92 @@
+# Azure Core OpenTelemetry Tracing
```

Comment:
If we're going to ship this, may as well spruce up the README - in a separate PR, perhaps, and maybe @ronniegeraghty can help out based on other languages' READMEs - for a shipping crate. I think this is fine for this PR, though: important info and examples.